snudown.c

```diff
@@ -10,6 +10,7 @@
 enum snudown_renderer_mode {
```

Comment:
Exactly same code is in fuzzing/snudown-validator.c  

I don't understand how fuzzer/validator works in this code base, yet, but that seems like a DRY violation?